src/map/npc.cpp

```diff
@@ -4956,7 +4956,7 @@ int npc_do_atcmd_event(struct map_session_data* sd, const char* command, const c
 
 	for( i = 0; i < ( strlen( message ) + 1 ) && k < 127; i ++ ) {
 		if( message[i] == ' ' || message[i] == '\0' ) {
-			if( message[ ( i - 1 ) ] == ' ' ) {
+			if ((i == 0 && message[i] == '\0') || (i > 0 && message[i - 1] == ' ')) {
```

Comment:
Why not simply "if i > 0 && ..."?

Reply:
I thought we would increment .@atcmd_numparameters if I didn't have that check, but you're right, we can just remove it.